roles/command_line_tools/tasks/bootstrap.yml

```
@@ -0,0 +1,45 @@
+- name: Check for Xcode Command Line Tools
+  raw: stat /Library/Developer/CommandLineTools
+  register: xcode_cl_tools
+  changed_when: no
+  failed_when: xcode_cl_tools.rc not in [0, 1]
+  tags:
+    - macos_command_line_tools
+    - macos
+    - xcode
+
+- name: Install command line tools
+  when: xcode_cl_tools.stderr is search("No such file or directory")
+  block:
+    - name: Create hidden install file
+      raw: touch {{ _macos_cli_inprogress_file }}
```

The docs recommend always quoting vars injected into `raw` invocations:
```suggestion
      raw: touch {{ _macos_cli_inprogress_file | quote }}
```